GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2733,16 +2735,15 @@ private void DiffContextMenu_Opening(object sender, System.ComponentModel.Cancel
                 artificialRevSelected = artificialRevSelected || selectedRevisions[selectedRevisions.Count - 1].IsArtificial();
 
             // disable items that need exactly one selected item
-            bool isExcactlyOneItemSelected = DiffFiles.SelectedItems.Count() == 1;
-            var isCombinedDiff = isExcactlyOneItemSelected &&
+            bool isExactlyOneItemSelected = DiffFiles.SelectedItems.Count() == 1;
+            var isCombinedDiff = isExactlyOneItemSelected &&
                 DiffFiles.CombinedDiff.Text == DiffFiles.SelectedItemParent;
-            var enabled = isExcactlyOneItemSelected && !isCombinedDiff;
```

Comment:
We can disable the menu if all the files selected are under the combined diff section, enable the menu if any of the selected file is not combined diff.